settings/common.py

```diff
@@ -282,6 +282,9 @@
 # For inviting users to participant and host teams.
 ADMIN_EMAIL = "admin@cloudcv.org"
 CLOUDCV_TEAM_EMAIL = "EvalAI Team <team@cloudcv.org>"
+SLACK_WEBHOOKS = {
+    'default': 'http://testslackwebhook.com/webhook'
```

Comment:
Let's not hardcode it. Can you please fetch this from the environment ?

Reply:
@deshraj I didn't understand what you meant by fetching from the environment. Could you please elaborate

Reply:
did you mean this (https://stackoverflow.com/questions/4906977/how-do-i-access-environment-variables-from-python) ?